server/certstore/certstore.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2022-2023 The NATS Authors
```

Comment:
how about moving this package to be under `internal/certstore` so that it is not exposed as public? That is where I put the ldap parsing things: https://github.com/nats-io/nats-server/tree/main/internal/ldap

Reply:
Hmm, do we want it exposed @tbeets ?

Only reason would be for possible tooling etc.